xml/admin_ceph_troubleshooting.xml

```diff
@@ -522,7 +522,7 @@ keep_jobs: 1</screen>
   <title>OSD Panic Occurs when Media Error Happens during Directory Split</title>
 
   <para>
-   When directory split error is generated, the corresponding OSD is designed
+   When directory split error is generated on &filestore; OSDs, the corresponding OSD is designed
```

Comment:
:+1:  I still would like the filestore is mentioned in the title.

Reply:
done